internal/errors.go

```diff
@@ -0,0 +1,9 @@
+package internal
+
+type FriendlyError struct {
```

Comment:
```suggestion
type UserError struct {
```

(and the field inside can be called `Message`)

"FriendlyError.UserError" sounds like weird naming - I think the more important part to convey as part of the name is that this is a user error, not an internal error.